fastmigrate/core.py

```diff
@@ -462,22 +460,22 @@ def run_migrations(
             # Start timing this migration
             migration_start = time.time()
             if verbose:
-                console.print(f"[blue]Applying[/blue] migration [bold]{version}[/bold]: [cyan]{script_name}[/cyan]")
+                print(f"Applying migration {version}: {script_name}")
             
             # Each script will open its own connection
             
             # Execute the migration script
             success = execute_migration_script(db_path, script_path)
             
             if not success:
-                console.print(f"[bold red]Migration failed:[/bold red] {script_path}")
+                sys.stderr.write(f"Migration failed: {script_path}\n")
                 stats["failed"] += 1
                 
                 # Show summary of failure - always show errors regardless of verbose flag
-                console.print("\n[bold red]Migration Failed[/bold red]")
-                console.print(f"  • [bold]{stats['applied']}[/bold] migrations applied")
-                console.print(f"  • [bold]{stats['failed']}[/bold] migrations failed")
-                console.print(f"  • Total time: [bold]{time.time() - start_time:.2f}[/bold] seconds")
+                sys.stderr.write("\nMigration Failed\n")
```

Comment:
Mixing stderr and stdout in one message gets a bit confusing, since they're separate streams. Use the same stream for a single message. This kind of thing is really what multiline strings are for! :)